code/style.css

```diff
@@ -1,3 +1,358 @@
-body {
-  background: #FFECE9;
-}
\ No newline at end of file
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500&family=Roboto&display=swap");

```

Comment:
I usually put the link to the fonts in the head of the html. Is there a specific reason for why you put it in the stylesheet or is it just a preference?